api/ibmcapi/types.go

```diff
@@ -0,0 +1,15 @@
+// Types copied from https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/ba09aa01b5f23d13917fc455b9e6aaa885809e23/api/v1beta2/ibmpowervsmachine_types.go
```

Comment:
I'm trying to understand this. Why copy these rather than use them straight from IBM CAPI? Just so we don't have to import the whole dep?

Reply:
Yep, the main point is make the API lighter, as much as we can.